seed_test_data errors

### Bug Description

Running `nox -s seed_test_data` throws the following stack trace

```console
⠿ fides-ui Error                                                                                                                               0.6s
Error response from daemon: manifest for ethyca/fides:local-ui not found: manifest unknown: manifest unknown
nox > Session seed_test_data raised exception Exception('Error executing command: docker compose -f docker-compose.yml -f docker/docker-compose.integration-postgres.yml up -d')
Traceback (most recent call last):
  File "/Users/paul/development/python/fides/.venv/lib/python3.9/site-packages/nox/sessions.py", line 728, in execute
    self.func(session)
  File "/Users/paul/development/python/fides/.venv/lib/python3.9/site-packages/nox/_decorators.py", line 75, in __call__
    return self.func(*args, **kwargs)
  File "/Users/paul/development/python/fides/noxfiles/utils_nox.py", line 33, in seed_test_data
    run_infrastructure(datastores=["postgres"], run_create_test_data=True)
  File "/Users/paul/development/python/fides/noxfiles/run_infrastructure.py", line 98, in run_infrastructure
    return _run_create_test_data(path, COMPOSE_SERVICE_NAME)
  File "/Users/paul/development/python/fides/noxfiles/run_infrastructure.py", line 169, in _run_create_test_data
    _run_cmd_or_err(f"docker compose {path} up -d")
  File "/Users/paul/development/python/fides/noxfiles/run_infrastructure.py", line 146, in _run_cmd_or_err
    raise Exception(f"Error executing command: {cmd}")
Exception: Error executing command: docker compose -f docker-compose.yml -f docker/docker-compose.integration-postgres.yml up -d
nox > Session seed_test_data failed.
```

#### Steps to Reproduce

1. Run `nox -s "build(dev)"`
1. Run `nox -s seed_test_data`

### Expected behavior

Should run without error.

### Screenshots

_If applicable, add screenshots to help explain your problem._

### Environment

* Version: unified fides
* OS:
* Python Version:
* Docker Version:

### Additional context

_Add any other context about the problem here._

`init_db` has been removed from the `ops` directory only leaving `init_db` from `ctl`. Switching this in `create_test_data.py` removes the above error, but creates a new one.

```console
2022-09-30 16:14:33.817 | INFO     | fides.api.ctl.database.database:init_db:49 - Initializing database
2022-09-30 16:14:33.826 | INFO     | fides.api.ctl.database.database:upgrade_db:41 - Running database migrations
2022-09-30 16:14:34.438 [INFO] (database:load_default_taxonomy:72): Loading the default fideslang taxonomy...
2022-09-30 16:14:34.439 [INFO] (database:load_default_taxonomy:74): Processing organization resources...
2022-09-30 16:14:34.457 [DEBUG] (crud:get_resource:67): Fetching resource | {'sql_model': 'Organization', 'fides_key': 'default_organization'}
2022-09-30 16:14:34.939 [ERROR] (crud:create_resource:38): Failed to insert resource | {'sql_model': 'Organization', 'fides_key': 'default_organization', 'error': 'resource already exists'}
2022-09-30 16:14:34.940 [INFO] (database:load_default_taxonomy:84): INSERTED 0 organization resource(s)
2022-09-30 16:14:34.941 [INFO] (database:load_default_taxonomy:85): SKIPPED 1 organization resource(s)
2022-09-30 16:14:34.942 [INFO] (database:load_default_taxonomy:90): INSERTING new default fideslang taxonomy resources
2022-09-30 16:14:34.942 [INFO] (database:load_default_taxonomy:92): Processing data_use resources...
2022-09-30 16:14:34.959 [DEBUG] (crud:list_resource:94): Fetching resources | {'sql_model': 'DataUse'}
2022-09-30 16:14:34.974 [INFO] (database:load_default_taxonomy:103): No new data_use resources to add from default taxonomy.
2022-09-30 16:14:34.975 [INFO] (database:load_default_taxonomy:92): Processing data_qualifier resources...
2022-09-30 16:14:34.991 [DEBUG] (crud:list_resource:94): Fetching resources | {'sql_model': 'DataQualifier'}
2022-09-30 16:14:35.002 [INFO] (database:load_default_taxonomy:103): No new data_qualifier resources to add from default taxonomy.
2022-09-30 16:14:35.003 [INFO] (database:load_default_taxonomy:92): Processing data_category resources...
2022-09-30 16:14:35.020 [DEBUG] (crud:list_resource:94): Fetching resources | {'sql_model': 'DataCategory'}
2022-09-30 16:14:35.036 [INFO] (database:load_default_taxonomy:103): No new data_category resources to add from default taxonomy.
2022-09-30 16:14:35.037 [INFO] (database:load_default_taxonomy:92): Processing data_subject resources...
2022-09-30 16:14:35.053 [DEBUG] (crud:list_resource:94): Fetching resources | {'sql_model': 'DataSubject'}
2022-09-30 16:14:35.066 [INFO] (database:load_default_taxonomy:103): No new data_subject resources to add from default taxonomy.
Seeding database with privacy requests
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1802, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 719, in do_execute
    cursor.execute(statement, parameters)
psycopg2.errors.NotNullViolation: null value in column "name" violates not-null constraint
DETAIL:  Failing row contains (pol_4f68f262-37c2-4d13-8551-28b12034e4e5, 2022-09-30 16:14:35.10021+00, 2022-09-30 16:14:35.10021+00, null, delete, null, null, null).


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/fides/scripts/create_test_data.py", line 276, in <module>
    create_test_data(session)
  File "/fides/scripts/create_test_data.py", line 172, in create_test_data
    _create_policy(
  File "/fides/scripts/create_test_data.py", line 48, in _create_policy
    created, policy = Policy.get_or_create(
  File "/usr/local/lib/python3.10/site-packages/fideslib/db/base_class.py", line 214, in get_or_create
    db_obj = cls.create(db=db, data=data)  # type: ignore
  File "/usr/local/lib/python3.10/site-packages/fideslib/db/base_class.py", line 175, in create
    return cls.persist_obj(db, db_obj)
  File "/usr/local/lib/python3.10/site-packages/fideslib/db/base_class.py", line 286, in persist_obj
    db.commit()
  File "/usr/local/lib/python3.10/site-packages/fideslib/db/session.py", line 63, in commit
    return super().commit()
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 1428, in commit
    self._transaction.commit(_to_root=self.future)
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 829, in commit
    self._prepare_impl()
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 808, in _prepare_impl
    self.session.flush()
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 3345, in flush
    self._flush(objects)
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 3484, in _flush
    with util.safe_reraise():
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
    raise exception
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/session.py", line 3445, in _flush
    flush_context.execute()
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/unitofwork.py", line 456, in execute
    rec.execute(self)
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/unitofwork.py", line 630, in execute
    util.preloaded.orm_persistence.save_obj(
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/persistence.py", line 244, in save_obj
    _emit_insert_statements(
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/persistence.py", line 1221, in _emit_insert_statements
    result = connection._execute_20(
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1614, in _execute_20
    return meth(self, args_10style, kwargs_10style, execution_options)
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/sql/elements.py", line 325, in _execute_on_connection
    return connection._execute_clauseelement(
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1481, in _execute_clauseelement
    ret = self._execute_context(
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1845, in _execute_context
    self._handle_dbapi_exception(
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 2026, in _handle_dbapi_exception
    util.raise_(
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
    raise exception
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py", line 1802, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/default.py", line 719, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.IntegrityError: (psycopg2.errors.NotNullViolation) null value in column "name" violates not-null constraint
DETAIL:  Failing row contains (pol_4f68f262-37c2-4d13-8551-28b12034e4e5, 2022-09-30 16:14:35.10021+00, 2022-09-30 16:14:35.10021+00, null, delete, null, null, null).

[SQL: INSERT INTO policy (id, name, key, drp_action, execution_timeframe, client_id) VALUES (%(id)s, %(name)s, %(key)s, %(drp_action)s, %(execution_timeframe)s, %(client_id)s)]
[parameters: {'id': 'pol_4f68f262-37c2-4d13-8551-28b12034e4e5', 'name': None, 'key': 'delete', 'drp_action': None, 'execution_timeframe': None, 'client_id': None}]
(Background on this error at: https://sqlalche.me/e/14/gkpj)
nox > Session seed_test_data raised exception Exception('Error executing command: docker compose run fides python scripts/create_test_data.py')
Traceback (most recent call last):
  File "/Users/paul/development/python/fides/.venv/lib/python3.9/site-packages/nox/sessions.py", line 728, in execute
    self.func(session)
  File "/Users/paul/development/python/fides/.venv/lib/python3.9/site-packages/nox/_decorators.py", line 75, in __call__
    return self.func(*args, **kwargs)
  File "/Users/paul/development/python/fides/noxfiles/utils_nox.py", line 33, in seed_test_data
    run_infrastructure(datastores=["postgres"], run_create_test_data=True)
  File "/Users/paul/development/python/fides/noxfiles/run_infrastructure.py", line 98, in run_infrastructure
    return _run_create_test_data(path, COMPOSE_SERVICE_NAME)
  File "/Users/paul/development/python/fides/noxfiles/run_infrastructure.py", line 170, in _run_create_test_data
    _run_cmd_or_err(
  File "/Users/paul/development/python/fides/noxfiles/run_infrastructure.py", line 146, in _run_cmd_or_err
    raise Exception(f"Error executing command: {cmd}")
Exception: Error executing command: docker compose run fides python scripts/create_test_data.py
nox > Session seed_test_data failed.
```

--- a/scripts/create_test_data.py
+++ b/scripts/create_test_data.py
@@ -23,7 +23,8 @@ from fides.api.ops.models.privacy_request import (
     PrivacyRequest,
     PrivacyRequestStatus,
 )
-from fides.api.ops.models.storage import ResponseFormat, StorageConfig
+from fides.api.ops.schemas.storage.storage import ResponseFormat
+from fides.api.ops.models.storage import StorageConfig
 from fides.api.ops.schemas.redis_cache import Identity
 from fides.api.ops.schemas.storage.storage import (
     FileNaming,

--- a/scripts/create_test_data.py
+++ b/scripts/create_test_data.py
@@ -1,5 +1,5 @@
 """Script to create test data for the Admin UI"""
-
+import asyncio
 import string
 from datetime import datetime, timedelta
 from uuid import uuid4
@@ -10,8 +10,7 @@ from fideslib.models.client import ClientDetail
 from fideslib.models.fides_user import FidesUser
 from sqlalchemy import orm
 
-from fides.ctl.core.config import get_config
-from fides.api.ops.db.database import init_db
+from fides.api.ctl.database.database import init_db
 from fides.api.ops.models.connectionconfig import (
     AccessLevel,
     ConnectionConfig,
@@ -32,6 +31,9 @@ from fides.api.ops.schemas.storage.storage import (
     StorageType,
 )
 from fides.api.ops.util.data_category import DataCategory
+from fides.ctl.core.config import get_config
+
+CONFIG = get_config()
 
 
 def _create_policy(
@@ -268,7 +270,7 @@ def create_test_data(db: orm.Session) -> FidesUser:
 
 
 if __name__ == "__main__":
-    init_db(CONFIG.database.sqlalchemy_database_uri)
+    asyncio.run(init_db(CONFIG.database.sqlalchemy_database_uri))
     session_local = get_db_session(CONFIG)
     with session_local() as session:
         create_test_data(session)

--- a/scripts/create_test_data.py
+++ b/scripts/create_test_data.py
@@ -46,19 +46,15 @@ def _create_policy(
     """
     Util method to create policies
     """
-    created, policy = Policy.get_or_create(
-        db=db,
-        data={
-            "key": policy_key,
-        },
-    )
+    policy = Policy.get_by(db=db, field="key", value=policy_key)
 
-    if not created:
+    if policy:
         # If the Policy is already created, don't create it again
         return policy
     else:
-        policy.client_id = client_id
-        policy.name = policy_key
+        policy = Policy.create(
+            db=db, data={"client_id": client_id, "name": policy_key, "key": policy_key}
+        )
         policy.save(db=db)
 
     rand = string.ascii_lowercase[:5]

--- a/src/fides/api/ops/models/connectionconfig.py
+++ b/src/fides/api/ops/models/connectionconfig.py
@@ -2,7 +2,7 @@ from __future__ import annotations
 
 import enum
 from datetime import datetime
-from typing import TYPE_CHECKING, Any, Dict, Optional, Type
+from typing import Any, Dict, Optional, Type
 
 from fideslib.db.base import Base
 from fideslib.db.base_class import get_key_from_data
@@ -20,8 +20,6 @@ from fides.api.ops.db.base_class import JSONTypeOverride
 from fides.api.ops.schemas.saas.saas_config import SaaSConfig
 from fides.ctl.core.config import get_config
 
-if TYPE_CHECKING:
-    from fides.api.ops.models.manual_webhook import AccessManualWebhook
 CONFIG = get_config()
 
 
@@ -125,7 +123,7 @@ class ConnectionConfig(Base):
         MutableDict.as_mutable(JSONB), index=False, unique=False, nullable=True
     )
 
-    access_manual_webhook = relationship(
+    access_manual_webhook = relationship(  # type: ignore[misc]
         "AccessManualWebhook",
         back_populates="connection_config",
         cascade="delete",

--- a/src/fides/api/ops/models/manual_webhook.py
+++ b/src/fides/api/ops/models/manual_webhook.py
@@ -26,7 +26,7 @@ class AccessManualWebhook(Base):
         nullable=False,
     )
     connection_config = relationship(
-        "ConnectionConfig", back_populates="access_manual_webhook", uselist=False
+        ConnectionConfig, back_populates="access_manual_webhook", uselist=False
     )
 
     fields = Column(MutableList.as_mutable(JSONB), nullable=False)

--- a/src/fides/api/ops/task/graph_task.py
+++ b/src/fides/api/ops/task/graph_task.py
@@ -7,7 +7,7 @@ from time import sleep
 from typing import Any, Callable, Dict, List, Optional, Set, Tuple, Union
 
 import dask
-from dask import delayed
+from dask import delayed  # type: ignore[attr-defined]
 from dask.threaded import get
 from sqlalchemy.orm import Session
 

